source/shared/net/desktop-wallet/account-recovery.rst

```diff
@@ -0,0 +1,41 @@
+.. _account-recovery-desktop:
+
+=====================
+Account recovery
+=====================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Account recovery in the desktop wallet
+=======================================================
+
+The recovery process is used to recover lost accounts using only the ledger device, which was used to create those accounts.
+
+It should only be necessary to perform this recovery if you have lost the backup of your accounts and identities. To learn more about making backups of your data, see :ref:`Make a backup of identities, accounts, and addresses <export-import-desktop>`.
+
+If the user has lost their ledger device or the device has ceased to function, the keys can be restored in another ledger device, by initialising it with the same recovery phrase as the original device. To learn more about the recovery phrase of a ledger device:
+`What is a recovery phrase <https://www.ledger.com/academy/crypto/what-is-a-recovery-phrase/>`_ .
+
+   .. Note::
+      This recovery process will recover your accounts, but it is not possible to recover the identities used to create these accounts, because they do not exists on the chain, to learn more about identities see :ref:`Identities and accounts <reference-id-accounts>`.
```

Comment:
```suggestion
      This recovery process only recovers your accounts, not the identities used to create the accounts because they don't exist on the chain. To learn more about identities, see :ref:`Identities and accounts <reference-id-accounts>`.
```